### PR TITLE
Merge from streamr-client-javascript

### PR DESCRIPTION
**Pull request by jtakalai**

### CODE REVIEW

File: packages/client/README.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
   </a>
 </p>
 <h1 align="left">
-  Streamr JavaScript Client
+  <strike> Streamr JavaScript Client </strike>
```
Comment (timoxley) [inline packages/client/README.md]: This probably shouldn't be here

File: packages/client/test/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    rules: {
+        'import/no-extraneous-dependencies': 'off', // otherwise will complain that tests shouldn't contain devDependencies...
```
Comment (timoxley) [inline packages/client/test/.eslintrc.js]: ?? I've never seen this

File: packages/client/test/integration/EncryptionKeyPersistence.test.ts [hunk 3]
```diff
@@ -0,0 +1,278 @@
+import { wait } from 'streamr-test-utils'
```
Comment (timoxley) [inline packages/client/test/integration/EncryptionKeyPersistence.test.ts]: The addition of this file seems weird.

File: packages/client/.husky/commit-msg [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+BRANCH_NAME=$(git symbolic-ref --short -q HEAD)
+
+if [ $BRANCH_NAME != '(no branch)' ]
+then
+    # don't run hook when rebasing
+    . "$(dirname "$0")/_/husky.sh"
+
+    npx --no-install commitlint --edit
+fi
```
Comment (timoxley) [inline packages/client/.husky/commit-msg]: I don't think this gets used anymore, there's a top-level `.husky`.

File: packages/client/test/.eslintrc.js (hunk 2 above)
Comment (jtakalai) [reply packages/client/test/.eslintrc.js]: maybe it was something else causing it

**Review by timoxley — APPROVED** [review]
lgtm, failing tests are the known long resend test failure

**Comment by jtakalai** [comment]
ok, only red tests left that are red in main, too. Merging.

**Pull request merged**